README.md

```diff
@@ -20,26 +25,36 @@ What this means is that unlike many other Prettier plugins,
 `prettier-java` has **no additional runtime pre-requisites** (e.g: Python executable).
 It could even be used inside a browser.
 
+## Subpackages
+
+This project contains 2 packages:
```

Comment:
May be worth experimenting with add npm icon links next to each package name.
You can get those links/imgs from: https://shields.io/category/version